src/MoBi.Presentation/DTO/ContainerDTO.cs

```diff
@@ -17,5 +17,10 @@ public ContainerDTO(IContainer container) : base(container)
       public string ParentPath { get; set; }
 
       public bool ParentPathEditable { get; set; }
+
+      public void AddModuleName(string moduleName)
+      {
+         Name = $"{moduleName} - {Name}";
```

Comment:
That's a bit weird. The name of the Container is not the name of the underlying object anymore
Feels more like we need a display name property or sthg